.hound.yml

```diff
@@ -0,0 +1,5 @@
+fail_on_violations: true
+
+eslint:
+  enabled: true
+  config_file: .eslintrc.json
```

Comment:
I think it can be better if we enable `sass` checking here and also ignoring `.eslintignore` file

Reply:
I don't think so, because this means we are ignoring the eslintignore file while we need it to ignore the file that we don't need to lint with Eslint